zstacklib/zstacklib/utils/ovn.py

```
@@ -119,7 +119,7 @@ def delVnicFromOvsByVmUuidIfExist(vmUuid, brName="br-int"):
 
             bash.bash_o(CtlBin + '--if-exists del-port {} {}'.format(brName, vnic_name))
     except Exception as err:
-        logger.exception("Delete vnic for brdige {} failed. {}".format(brName, err))
+        logger.exception("Delete vnic for bridge {} failed. {}".format(brName, err))
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)